src/components/Sidebar/Properties/PropertyOwnerHeader.jsx

```diff
@@ -39,6 +87,39 @@ function PropertyOwnerHeader({
     evt.stopPropagation();
   };
 
+  const onToggleCheckboxClick = (shouldBeEnabled, event) => {
+    if (!enabledUri) return;
+
+    const holdingShift = event.getModifierState('Shift');
+    const shouldNotFade = !fadeUri || (fadeDuration < 0.001) || holdingShift;
+
+    if (shouldNotFade) {
+      getPropertyDispatcher(enabledUri).set(shouldBeEnabled);
+      // Also set the fade value to 1
+      getPropertyDispatcher(fadeUri).set(1.0);
+      setFadeDirection(0);
+      return;
+    }
+
+    const isFadingIn = fadeDirection > 0;
+    const isFadingOut = fadeDirection < 0;
+    const shouldFadeIn = isFadingOut || (shouldBeEnabled && !isFadingIn);
+
+    // If fade in, first set fade value to 0 to make sure it's fully hidden
```

Comment:
This is a comment that I have forgotten to remove :) 

But actually, there is no reason anymore (there was when I first implemented it). I'll remove both the comment and the check below

Reply:
Nevermind! There is a reason. We might click the checkbox for an item that is disabled, but has a fade value of 1 already. This would not result in any fading, unless we first set the fade value to zero.